docs/Reference/API-Methods.md

```diff
@@ -389,10 +389,11 @@ Properties of the remote node object are:
   address might not match the hex value for `port`. The remote address depends on which node
   initiated the connection.
 * `port` - Port on the remote node on which P2P discovery is listening.
-* `id` - Node public key. Excluding the `0x` prefix, the node public key is the ID in the enode
-  URL `enode://<id ex 0x>@<host>:<port>`.
+* `id` - Node public key. Excluding the `0x` prefix, the node public key is the ID in the [enode
+  URL](../Concepts/Node-Keys.md#enode-url) `enode://<id ex 0x>@<host>:<port>`.
```

Comment:
It would be safer to have the whole link text on the same line to work properly on all Markdown tools.